lib/crypto/src/curve/mod.rs

```diff
@@ -236,15 +236,26 @@ pub trait AffineRepr:
     }
 }
 
-/// Given a vector of field elements `v_i`, compute the vector `v_i^(-1)`
+/// Efficiently computes inverses of non-zero elements in the slice.
+///
+/// Uses Montgomery's trick to compute multiple inverses with fewer field operations.
+/// Zero elements remain unchanged.
+///
+/// # Arguments
+///
+/// * `v` - Mutable slice of field elements for in-place inversion
```

Comment:
```suggestion
/// * `v` - Mutable slice of field elements for in-place inversion.
```
nit: add dots to ends of list items for consistency with other existing docs